libs/build_mapl.sh

```diff
@@ -73,5 +75,5 @@ VERBOSE=$MAKE_VERBOSE make -j${NTHREADS:-4} install
 
 # generate modulefile from template
 modpath=mpi
-$MODULES && update_modules $modpath $name $id
+$MODULES && update_modules $modpath $name $install_as "" " -DMAPL_ESMF_VERSION=${ESMF_VERSION}"
```

Comment:
I like this approach.
1 suggestion:
```suggestion
$MODULES && update_modules $modpath $name $install_as "" "-DMAPL_ESMF_VERSION=${ESMF_VERSION}"
```